src/org/daisy/dotify/formatter/impl/sheet/VolumeImpl.java

```diff
@@ -49,15 +64,18 @@ public int getBodySize() {
         return bodyVolSize;
     }
 
+    /**
+     * The actual size of the whole volume in sheets.
+     */
     public int getVolumeSize() {
```

Comment:
```
> Task :javadoc
/home/danielp/github/dotify.library/src/org/daisy/dotify/formatter/impl/sheet/VolumeImpl.java:70: warning: no @return
    public int getVolumeSize() {
               ^
1 warning
```

I know it's just a warning but if we only have one documentation issue it could be good to rectify before we continue.

Reply:
Warning fixed